testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1685,3 +1685,102 @@
   node = get_target(host)
   node.run_until_ok("lsof  -i:#{port}")
 end
+
+Then(/^port "([^"]*)" should be (open|closed)$/) do |port, selection|
+  _output, code = $server.run("ss --listening --numeric | grep :#{port}", check_errors: false, verbose: true)
+  port_opened = code.zero?
+  if selection == 'closed'
+    raise "Port '#{port}' open although it should not be!" if port_opened
+  else
+    raise "Port '#{port}' not open although it should be!" unless port_opened
+  end
+end
+
+# rebooting via SSH
+When(/^I reboot the server through SSH$/) do
+  init_string = "ssh:#{$server.public_ip}"
+  temp_server = twopence_init(init_string)
+  temp_server.extend(LavandaBasic)
+  temp_server.run('reboot > /dev/null 2> /dev/null &')
+  default_timeout = 300
+
+  check_shutdown($server.public_ip, default_timeout)
+  check_restart($server.public_ip, temp_server, default_timeout)
+
+  repeat_until_timeout(timeout: default_timeout, message: "Spacewalk didn't come up") do
+    out, code = temp_server.run('spacewalk-service status', check_errors: false, timeout: 10)
+    if !out.to_s.include? "dead" and out.to_s.include? "running"
+      log "Server spacewalk service is up"
+      break
+    end
+    sleep 1
+  end
+end
+
+When(/^I reboot the "([^"]*)" minion through SSH$/) do |host|
+  node = get_target(host)
+  node.run('reboot > /dev/null 2> /dev/null &')
+  reboot_timeout = 120
+  check_shutdown($node.public_ip, reboot_timeout)
+  check_restart($server.public_ip, node, reboot_timeout)
+end
+
+When(/^I reboot the "([^"]*)" minion through the web UI$/) do |host|
+  steps %(
+
+  Given I am on the Systems overview page of this "#{host}"
+  When I follow first "Schedule System Reboot"
+  Then I should see a "System Reboot Confirmation" text"
+  And I should see a "Reboot system" button"
+  When I click on "Reboot system"
+  Then I should see a "Reboot scheduled for system" text"
+  And I wait at most 600 seconds until event "System reboot scheduled by admin" is completed"
+  Then I should see a "This action's status is: Completed" text"
+  )
```

Comment:
The bracket has to be moved further to the ringt to be aligned with the starting one. See original code.

Reply:
I think the bracket is right the stuff inside needed to be moved though

Reply:
When I changed this step, RuboCop complained about the bracket. But I guess it is fine. The content should stay at this level.